plutus-benchmark/marlowe/src/PlutusBenchmark/Marlowe/BenchUtil.hs

```diff
@@ -92,6 +90,60 @@ readBenchmark filename =
                pure Benchmark{..}
         _ -> Left "Failed deserializing benchmark file."
 
+-- Rewrite all of a particular script hash in the script context.
+updateScriptHash
+  :: ScriptHash
+  -> ScriptHash
+  -> ScriptContext
+  -> ScriptContext
+updateScriptHash oldHash newHash scriptContext =
```

Comment:
if ever there was a case for lenses it's this monstrosity :scream:

Reply:
Indeed!